doc/version-history.rst

```diff
@@ -8,6 +8,12 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+Cycle 0019
+==========
+
+* Release of XML 9.0.0
+* Updated the Dockerfiles of ptg and scriptqueue
```

Comment:
Can you describe all the other changes you made as well? 

* Removed unnecessary container `aos_aoclc`.
* Updated conda-builder for arch64
* Update deploy-env for arch64

Reply:
Done